packages/expo-in-app-purchases/android/src/main/java/expo/modules/inapppurchases/BillingManager.java

```diff
@@ -416,7 +423,7 @@ public static Bundle purchaseToBundle(Purchase purchase) {
 
     bundle.putBoolean("acknowledged", purchase.isAcknowledged());
     bundle.putString("orderId", purchase.getOrderId());
-    bundle.putString("productId", purchase.getSku());
+    bundle.putStringArrayList("productIds", purchase.getSkus());
```

Comment:
To avoid breaking API changes, we could implement a `productId` getter raising a deprecated warning (but still returning the first productId).

Reply:
Yes
I'm not sure whether we should return first productId or add productdIds to iOS.

Reply:
I'd favour productIds everywhere to simplify the types